src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIStreams.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Data.SqlClient.SNI
     /// <summary>
```

Comment:
There are two  unused libraries at the top System.Threading and System.Threading.Tasks can you remove them please?

Reply:
Done.